src/main/scala/cromwell/engine/io/gcs/GcsFileAttributes.scala

```diff
@@ -0,0 +1,25 @@
+package cromwell.engine.io.gcs
+
+import java.nio.file.Path
+import java.nio.file.attribute.{FileTime, BasicFileAttributes}
+
+class GcsFileAttributes(path: Path) extends BasicFileAttributes {
+
+  override def fileKey(): AnyRef = ???
+
+  override def isRegularFile: Boolean = ???
+
+  override def isOther: Boolean = ???
+
+  override def lastModifiedTime(): FileTime = ???
+
+  override def size(): Long = ???
+
+  override def isDirectory: Boolean = false
+
+  override def isSymbolicLink: Boolean = false
+
+  override def creationTime(): FileTime = ???
+
+  override def lastAccessTime(): FileTime = ???
```

Comment:
Nit pick:

Imho
```scala
???
```

[means](https://github.com/scala/scala/blob/v2.11.7/src/library/scala/NotImplementedError.scala#L13-L16)
```scala
// The author of this trait/class forgot to fill in the template file in before committing
```
not
```scala
throw new scala.NotImplementedError("This class did not end up implementing a semi-abstract method.")
```
nor the weaker
```scala
throw new java.lang.UnsupportedOperationException("Whoops, who called this?")
```